src/SearchField/README.md

```diff
@@ -0,0 +1,88 @@
+# SearchField
+
+Provides a basic search field component.
+
+## API
+
+### `screenReaderText` (string; required)
+`screenReaderText` specifies the text for screen readers within the `label` element of the `InputText` component.
+
+### `onSubmit` (function; required)
+`onSubmit` specifies a callback function for when the `SearchField` is submitted by the user. For example:
+
+```jsx
+<SearchField
+  screenReaderText="search"
+  onSubmit={(value) => { console.log(value); }}
+/>
+```
+
+### `onChange` (function; optional)
+`onChange` specifies a callback function for when the value in `SearchField` is changed by the user. The default is an empty function. For example:
+
+```jsx
+<SearchField
+  screenReaderText="search"
+  onChange={(value) => { console.log(value); }}
+/>
+```
+
+### `onFocus` (function; optional)
+`onFocus` specifies a callback function for when the value in `SearchField` is changed by the user. The default is an empty function. For example:
+
+```jsx
+<SearchField
+  screenReaderText="search"
+  onFocus={(value) => { console.log(value); }}
+/>
+```
+
+### `onBlur` (function; optional)
+`onBlur` specifies a callback function for when the value in `SearchField` is changed by the user. The default is an empty function. For example:
+
+```jsx
+<SearchField
+  screenReaderText="search"
+  onBlur={(value) => { console.log(value); }}
+/>
+```
+
+### `placeholder` (string; optional)
+`placeholder` specifies the placeholder text for the input. The default is an empty string.
+
+### `value` (string; optional)
+`value` specifies the initial value for the input. The default is an empty string.
+
+### `hasClearButton` (boolean; optional)
+`hasClearButton` specifies whether the search field has an additional button to allow the user to clear the `input` value. The default is `true`.
+
+### `hasRoundedCorners` (boolean; optional)
```

Comment:
Where's the boundary between the CSS that the component comes with and the CSS that the user of the API wants?

Basically, what if I want `reallyRoundedCorners` or `somewhatRoundedCorners`?

Reply:
Yeah, this is something I was struggling with how to handle. I was thinking of making this prop accept a string so that the user can pass in a value like `sm`, `md`, or `lg` to change the amount of border radius. If I go that route, I think it would also make sense for the prop to accept a boolean like it does now, but if no string is passed in, it defaults to Bootstrap's rounded corners.